src/components/TableBody.js

```diff
@@ -240,6 +240,11 @@ class TableBody extends React.Component {
                     fixedHeader={options.fixedHeader}
                     checked={this.isRowSelected(dataIndex)}
                     expandableOn={options.expandableRows}
+                    // When rows are expandable and selectable, but this row isn't expandable, set this to enabled.
```

Comment:
This comment is probably out of date now that we no longer check for selectable, right?

Reply:
iuhh correct, I will delete the selectable part.

Reply:
Updated it! let me know what you think.